StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkLogin/LinkLoginFooterView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  LinkLoginFooterView.swift
+//  StripeFinancialConnections
+//
+//  Created by Mat Schmid on 2024-07-25.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripeUICore
+import UIKit
+
+class LinkLoginFooterView: HitTestView {
```

Comment:
you might have been able to use `PaneLayoutView.createFooterView` to do this same thing (not 100% sure) - or at least wrap that function with `LinkLoginFooterView` and save some code

Reply:
TIL `PaneLayoutView.createFooterView` - I'm gonna try to switch over to using this! Easier than maintaining a whole other footer view class!

Reply:
It worked great! Just pushed a commit to use `PaneLayoutView.createFooterView` and deletes `LinkLoginFooterView` 👍